README.md

```diff
@@ -437,6 +437,7 @@ Clients for commercial social platforms that had their API access cut off in a w
 - [HackUp](https://github.com/mdh34/hackup) - [Hacker News](https://news.ycombinator.com) client `#vala` `#gtk3` `#granite`.
 - [Imaginer](https://github.com/ImaginerApp/Imaginer) - AI image generator (DALL·E 2, Portrait Plus, Stable Diffusion, Custom Provider) interface `#python` `#gtk4` `#libadwaita`.
 - [Lobjur](https://github.com/ranfdev/Lobjur) - [lobste.rs](https://lobste.rs) client `#gjs` `#gtk4` `#libadwaita`.
+- [Newelle](https://github.com/qwersyk/Newelle) - Virtual assistant (based on GPT4All) that interacts with the system by running shell commands and execute Python code `#python` `#gtk4` `#libadwaita`.
```

Comment:
I don't think this is the right category for Newelle.

Reply:
I think Bavarder is alternative of Newelle, and it placed in this

Reply:
Bavarder is a font-end to chat bot services while Newelle provides more that a wrapper. In my opinion, Newelle deserves to be in its own category.